textdistance/libraries.py

```diff
@@ -166,10 +166,13 @@ class SameLengthTextLibrary(SameLengthLibrary, TextLibrary):
 reg = prototype.register
 
 alg = 'DamerauLevenshtein'
-reg(alg, LibraryBase('abydos.distance', 'DamerauLevenshtein', presets={}, attr='dist_abs'))
-reg(alg, LibraryBase('pyxdameraulevenshtein', 'damerau_levenshtein_distance'))
-reg(alg, TextLibrary('jellyfish', 'damerau_levenshtein_distance'))
-reg(alg, LibraryBase('rapidfuzz.distance.DamerauLevenshtein', 'distance'))
+reg(alg, LibraryBase(
+    'abydos.distance', 'DamerauLevenshtein', presets={}, attr='dist_abs',
+    conditions=dict(restricted=False),
+))
+reg(alg, LibraryBase('pyxdameraulevenshtein', 'damerau_levenshtein_distance', conditions=dict(restricted=True)))
+reg(alg, TextLibrary('jellyfish', 'damerau_levenshtein_distance', conditions=dict(restricted=False)))
+reg(alg, LibraryBase('rapidfuzz.distance.DamerauLevenshtein', 'distance', conditions=dict(restricted=False)))
```

Comment:
```suggestion
reg(alg, LibraryBase('rapidfuzz.distance.DamerauLevenshtein', 'distance', conditions=dict(restricted=False)))
reg(alg, LibraryBase('rapidfuzz.distance.OSA', 'distance', conditions=dict(restricted=True)))
```
since `v2.9.0` rapidfuzz has an implementation of the optimal string alignment distance as well.